doc/docs.md

```diff
@@ -1556,12 +1556,17 @@ libraries and include files for Windows and Linux. V will provide you with a lin
 V can be used as an alternative to Bash to write deployment scripts, build scripts, etc.
 
 The advantage of using V for this is the simplicity and predictability of the language, and
-cross-platform support. "V scripts" run on Unix-like systems as well as on Windows.
+cross-platform support. "V scripts" run on UNIX-like systems as well as on Windows.
 
-Use .vsh file extension. It will make all functions in the `os`
+You can also use the `.vsh` file extension. It will make all functions in the `os`
 module global (so that you can use `ls()` instead of `os.ls()`, for example).
 
 ```v
+#!/usr/bin/env v run
```

Comment:
Unfortunately this does not work. v scripts are mostly useful when they can receive arguments on the CLI themselves. To do that though, v still needs the `run` command to know to run the .vsh script and pass everything after it to the script program, otherwise without `run`, v will try to interpret the arguments after the .vsh name as v options and will fail with an error message.

Using `#!/usr/bin/env v run` will try to find 'v run' as a program, which will fail (that is a limitation of the env program, not to v itself).

Recent versions of coreutils >=8.30, have env that has -S option, which allows you to do:
`#!/usr/bin/env -S v run` which works great, unfortunately coreutils 8.30 is from 2018, and there are some distros where it is not yet available :-| .

Given that on unix systems v supports `v symlink` which makes a symlink to /usr/local/bin/v ,
the most portable way to ensure that v scripts run is to use this shebang: `#!/usr/local/bin/v run`

This needs v to be symlinked in `/usr/local/bin/v` but works everywhere I have tried it, and works with `run` too, so that you can then pass arguments to your .vsh scripts without problems.

You can experiment for yourself with this program:
```v
#!/usr/local/bin/v run

for i, a in os.args {
  println('i: $i , arg: "$a"')
}
```

For me, with it:
```shell
0[14:23:32] /v/nv LINUX $ ./myscript.vsh a b c d 1234
new_parser: V script
["strings", "strconv.ftoa", "hash.wyhash", "strconv"]
["/v/nv/vlib/builtin/array.v", "/v/nv/vlib/builtin/builtin.v", "/v/nv/vlib/builtin/builtin_nix.v", "/v/nv/vlib/builtin/cfns.v", "/v/nv/vlib/builtin/float.v", "/v/nv/vlib/builtin/int.v", "/v/nv/vlib/builtin/map.v", "/v/nv/vlib/builtin/option.v", "/v/nv/vlib/builtin/sorted_map.v", "/v/nv/vlib/builtin/string.v", "/v/nv/vlib/builtin/utf8.v"]
i: 0 , arg: "/v/nv/myscript"
i: 1 , arg: "a"
i: 2 , arg: "b"
i: 3 , arg: "c"
i: 4 , arg: "d"
i: 5 , arg: "1234"
0[14:23:40] /v/nv LINUX $
```

Reply:
We can also change v to always run .vsh and ignore arguments after the .vsh file, even without `run`, so that `run` will not be needed. 
However, that is not yet done, and currently there is work underway to change the command line parsing stage, so it is better to do it after it, which will be in the comming 2-3 days, so for the time being I think you better change the shebang in the documentation to `#!/usr/local/bin/v run`

Reply:
Thanks for the detailed information :slightly_smiling_face: I'll make the required changes.